test/components/dragBoxLayerTests.ts

```diff
@@ -2,284 +2,241 @@
-
-      let startPoint = {
-        x: SVG_WIDTH / 4,
-        y: SVG_HEIGHT / 4
-      };
-      let endPoint = {
-        x: SVG_WIDTH / 2,
-        y: SVG_HEIGHT / 2
-      };
 
-      let target = dbl.background();
-      TestMethods.triggerFakeDragSequence(target, startPoint, endPoint);
-      assert.isTrue(dbl.boxVisible(), "box is drawn on drag");
-      let bounds = dbl.bounds();
-      assert.deepEqual(bounds.topLeft, startPoint, "top-left point was set correctly");
-      assert.deepEqual(bounds.bottomRight, endPoint, "bottom-right point was set correctly");
+    describe("DragBoxLayer - basics", () => {
+      let SVG_WIDTH = 400;
+      let SVG_HEIGHT = 400;
 
-      svg.remove();
-    });
-
-    it("dismisses on click", () => {
-      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      let dbl = new Plottable.Components.DragBoxLayer();
-      dbl.renderTo(svg);
+      let svg: d3.Selection<void>;
+      let dbl: Plottable.Components.DragBoxLayer;
+      let quarterPoint: Plottable.Point;
+      let halfPoint: Plottable.Point;
 
-      let targetPoint = {
-        x: SVG_WIDTH / 2,
-        y: SVG_HEIGHT / 2
-      };
+      beforeEach(() => {
+        svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+        dbl = new Plottable.Components.DragBoxLayer();
+        quarterPoint = {
+          x: SVG_WIDTH / 4,
+          y: SVG_HEIGHT / 4
+        };
+        halfPoint = {
+          x: SVG_WIDTH / 2,
+          y: SVG_HEIGHT / 2
+        };
+      });
 
-      let target = dbl.background();
-      TestMethods.triggerFakeDragSequence(target, targetPoint, targetPoint);
+      afterEach(() => {
+        svg.remove();
+      });
 
-      assert.isFalse(dbl.boxVisible(), "box is hidden on click");
+      it("correctly draws box on drag", () => {
+        dbl.renderTo(svg);
+        assert.isFalse(dbl.boxVisible(), "box is hidden initially");
 
-      svg.remove();
-    });
+        let target = dbl.background();
+        TestMethods.triggerFakeDragSequence(target, quarterPoint, halfPoint);
+        assert.isTrue(dbl.boxVisible(), "box is drawn on drag");
+        let bounds = dbl.bounds();
+        assert.deepEqual(bounds.topLeft, quarterPoint, "top-left point was set correctly");
+        assert.deepEqual(bounds.bottomRight, halfPoint, "bottom-right point was set correctly");
+      });
 
-    it("clipPath enabled", () => {
-      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      let dbl = new Plottable.Components.DragBoxLayer();
-      dbl.renderTo(svg);
-      TestMethods.verifyClipPath(dbl);
-      let clipRect = (<any> dbl)._boxContainer.select(".clip-rect");
-      assert.strictEqual(TestMethods.numAttr(clipRect, "width"), SVG_WIDTH, "the clipRect has an appropriate width");
-      assert.strictEqual(TestMethods.numAttr(clipRect, "height"), SVG_HEIGHT, "the clipRect has an appropriate height");
-      svg.remove();
-    });
+      it("dismisses on click", () => {
```

Comment:
I don't think the test matches the description
the dragbox was never drawn to begin with